ws-client/src/helpers.rs

```diff
@@ -72,6 +72,27 @@ pub fn process_subscription_response(
 	}
 }
 
+/// Attempts to process an incoming notification
+///
+/// Returns Ok() if the response was successfully handled
+/// Returns Err() if there was no handler for the method
+
+pub fn process_notification(manager: &mut RequestManager, notif: JsonRpcNotifResponse<JsonValue>) -> Result<(), Error> {
+	match manager.as_notification_handler_mut(&notif.method.to_owned()) {
+		Some(send_back_sink) => match send_back_sink.try_send(notif.params) {
+			Ok(()) => Ok(()),
+			Err(err) => {
+				log::error!("Error sending notification subscription {:?} error: {:?}", notif.method, err);
```

Comment:
An error here will be that the channel is full which means that the user couldn't keep with the notifications.

We should drop the notification to indicate this to the user, we do that for subscriptions.

Reply:
Then the user has re-register the method again to use it again or increase the size of buffer ^